test/test_autograd.py

```diff
@@ -295,28 +296,28 @@ def test_hessian_vector(self):
         z = x ** 2 + y * x + y ** 2
         z.backward(torch.ones(2, 2), create_graph=True)
 
-        x_grad = 2 * x.data + y.data
```

Comment:
Potentially we want these to run in no_grad mode, but that doesn't really change anything give that they are just used in the assertEqual below...